### ISSUE TITLE
Single place to see logs and traces across different applications

### ISSUE THREAD

**Issue by falgofrancis** [coroot/coroot#148]
HI,

We are trying to reduce observability cost and we find coroot to have very good capabilities. However I am not able to find a place where I can see logs or traces across different applications in one place. Is it in roadamp?

**Comment by def** [comment]
Indeed, we have plans to add a project-level overview for traces and logs

**Comment by anoop2811** [comment]
Hi @def , is this in your backlog and do you know if its tracked against a release?
